js/src/components/shipping-rate-section/minimum-order-card/minimum-order-input-label-text.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useCountryKeyNameMap from '.~/hooks/useCountryKeyNameMap';
+
+const firstN = 5;
+
+const MinimumOrderInputLabelText = ( props ) => {
+	const { countries } = props;
+	const keyNameMap = useCountryKeyNameMap();
+
+	const firstCountryNames = countries
+		.slice( 0, firstN )
+		.map( ( c ) => keyNameMap[ c ] );
+
+	const string =
+		countries.length > firstCountryNames.length
+			? __(
+					`Minimum order for <countries /> + <count /> more`,
+					'google-listings-and-ads'
+			  )
+			: __(
+					`Minimum order for <countries />`,
+					'google-listings-and-ads'
+			  );
+
+	return (
+		<div>
+			{ createInterpolateElement( string, {
+				countries: <strong>{ firstCountryNames.join( ', ' ) }</strong>,
+				count: <>{ countries.length - firstCountryNames.length }</>,
```

Comment:
💅  It would be better to use `sprintf` for dynamic values and only uses `createInterpolateElement` for wrapping HTML tags or react elements if it's possible.

Reply:
It could be done in this way:
https://github.com/woocommerce/google-listings-and-ads/blob/c492680956b0d0e198a4512a87527a41ea5e9134/js/src/components/paid-ads/budget-section/budget-recommendation/index.js#L32-L50

Reply:
Thanks for the example! Updated in [`4e9cfe9` (#1266)](https://github.com/woocommerce/google-listings-and-ads/pull/1266/commits/4e9cfe97b38524bd35f2146b5d0d2f6e99dd37ea), and it has a much better HTML output. 😃 👍